conf/machine/raspberrypi5.conf

```diff
@@ -24,7 +24,7 @@ RPI_KERNEL_DEVICETREE = " \
 SDIMG_KERNELIMAGE ?= "kernel_2712.img"
 SERIAL_CONSOLES ?= "115200;ttyAMA10"
 
-VC4DTBO ?= "vc4-kms-v3d"
+VC4DTBO ?= "vc4-kms-v3d-pi5"
```

Comment:
this is a common change that will affect wayland/weston or eglfs images as well. Can you see if core-image-weston boots for your case on pi5 ?

Reply:
I haven't needed this with either a pi5 or cm5 with I/O board. When did you need it?